unit_test/test_database.py

```diff
@@ -0,0 +1,4 @@
+from database import hello
+
+def test_hello():
+    assert hello() == "Hello world"
```

Comment:
You forgot to add a newline at the end of this file.
```suggestion
    assert hello() == "Hello world"
    
```